### DISCUSSION TITLE
How to request an update from Home assistant

### DISCUSSION THREAD

**Discussion by vezinpi** [zwave-js/zwave-js-ui#2222] [Q&A]
Hi,

I am using zwavejs2mqtt with home assistant. The MQTT part is not used.

I have a Radio thermostat CT100+ that has a problem providing the ThermostatOperatingState in a timely manner. Sometimes it may take more than 15 min before the State is updated. It was the case when it was installed on Vera and it is still the case since it is on HomeAssistant (zwaveJS + zwavejs2mqtt). The Home Assistant integration provide a way to refresh **all** values with SERVICE ZWAVE_JS.REFRESH_VALUE but that generate alot traffic on the network since I need to refresh every minute.

When using the MQTT GUI,  I can easily refresh the operating state by itself by browsing for the proper value.

My question:  

Is there a way to interact from HASS (or other) to ZWAVEJS2MQTT to request an update similar to what I can do from the GUI?

Thanks in advance

**Comment by kpine** [comment]
Enable polling. https://zwave-js.github.io/zwavejs2mqtt/#/device-support/polling?id=polling-legacy-z-wave-devices

**Reply by vezinpi** [reply]
This is great. Exactely what I needed.

I did not realised that refreshing a value was in fact the same as polling. Somehow I thought that polling was more for a whole device. I should have look closer to the MQTT manual.

Many thanks
